packages/cli/templates/csharp/echo/Echo.sln

```diff
@@ -0,0 +1,34 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio Version 17
+VisualStudioVersion = 17.0.31903.59
```

Comment:
NIT: Can we put all these magic strings somewhere common?

OTOH, I can also see this being simple copies

Reply:
those are auto gen from the dotnet cli, not sure of a way to make them common but I can look into it a bit.